src/coreclr/vm/callconvbuilder.cpp

```diff
@@ -320,7 +320,7 @@ HRESULT CallConv::TryGetUnmanagedCallingConventionFromModOpt(
     _In_ PCCOR_SIGNATURE pSig,
     _In_ ULONG cSig,
     _Inout_ CallConvBuilder* builder,
-    _Out_ UINT *errorResID)
+    _Out_ WORD *errorResID)
```

Comment:
We use `UINT` for resource IDs. For example, `RealCOMPlusThrowHR` that takes resource IDs has it as UINT. Why is this change necessary?

Reply:
Sad. This could have been done in a better way. Sorry. I will revert this.